### ISSUE TITLE
Model do not adapt to my dataset after finetuning

### ISSUE THREAD

**Issue by Shivam2JDS** [openai/CLIP#99]
After fine tuning the model on my dataset (containing 272 images) for 5 epoch and learning rate 5e-5, after training model just give same Image as result for every query.

**Comment by lonngxiang** [comment]
> After fine tuning the model on my dataset (containing 272 images) for 5 epoch and learning rate 5e-5, after training model just give same Image as result for every query.

similar situation

**Comment by MingYi0226** [comment]
Same issue

**Comment by terminator123** [comment]
have u solved the problem? i have the same problem
